tests/fast-integration/test/fixtures/getting-started-guide/index.js

```diff
@@ -131,7 +132,14 @@ async function ensureGettingStartedTestFixture() {
   await waitForServiceBinding(orderService, orderService);
   await k8sApply([sbuObj], orderService);
   const sbu = await waitForServiceBindingUsage(orderService, orderService);
-  await waitForPodWithSbuToBeReady(sbu);
+  const pod = await waitForPodWithSbuToBeReady(sbu);
```

Comment:
What if pod doesn't have the required env variables injected yet? I don't see retry here. Wouldn't it be better to return these env variables values directly from order-service? I see this `kubectl exec` is a little bit tricky.

Reply:
`waitForPodWithSbuToBeReady` checks if the pod has been labeled with sbu usage and if it's ready.
After that the envs has to be in.. no need to retry.. if they arent there its ok to fail the test.

Reply:
@pbochynski done. pls have a look